Src/VsVimShared/Implementation/Misc/FallbackKeyProcessor.cs

```diff
@@ -158,7 +177,7 @@ private bool TryConvertLetterToKeyInput(KeyEventArgs keyEventArgs, out KeyInput
             }
 
             var c = (char)('a' + (key - Key.A));
-            keyInput = KeyInputUtil.ApplyKeyModifiersToChar(c, keyModifiers);
+            keyInput = KeyInputUtil.ChangeKeyModifiersDangerous(KeyInputUtil.CharToKeyInput(c), keyModifiers);

```

Comment:
@jhamm it's unfortunately not easy to test unless you're very familiar with non-English keyboard layouts.  In the past I've had to rely on customers to walk me through the issues.  

VS makes this more difficult because it's notion of key bindings is equally odd and not always non-English friendly.  The `ChangeKeyModifiersDangerous` though is more along the lines of how VS thinks so it may be more appropriate here.  

I think we may just have to try this and see if it blocks anyone.